card/mod/solid_cache/set/abstract/solid_cache.rb

```diff
@@ -1,7 +1,7 @@
 # -*- encoding : utf-8 -*-
 
-# A card that includes Abstract::SolidCache caches its core view
-# in a '+*solid cache' card.
+# A card that includes Abstract::SolidCache has as its core view a cached version of
+# its "cacheable_core" view, which is storeed in a '+*solid cache' card.
```

Comment:
How can I cache the core view of a card with this approach?

Reply:
you could do `view :cacheable_core, :core`.

I'm open to other solutions, so long as we get rid of the view args.

btw, I'm thinking we move this out of decko core and into a mod.  Seems like a clean one.

Reply:
if you do `view :a, :b`, I don't think view a calls `render :b`; it defines view a using the block that defines view b.  You may have to specify the module where it's defined...

Reply:
I came up with a more decksterlicious solution.